# Specify 'allow-without' to allow --without-perl to be used
# TODO: Upstream

--- configure.ac.orig	2016-12-04 08:23:08 UTC
+++ configure.ac
@@ -373,7 +373,7 @@ GIT_PARSE_WITH_SET_MAKE_VAR(editor, DEFA
 GIT_ARG_SET_PATH(shell)
 #
 # Define PERL_PATH to provide path to Perl.
-GIT_ARG_SET_PATH(perl)
+GIT_ARG_SET_PATH(perl, allow-without)
 #
 # Define PYTHON_PATH to provide path to Python.
 GIT_ARG_SET_PATH(python, allow-without)
